testonly/hammer/reader.go

```diff
@@ -0,0 +1,145 @@
+package hammer
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"math/rand"
+	"reflect"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/google/trillian"
+	"github.com/google/trillian/client"
+	"github.com/google/trillian/testonly"
+	"github.com/google/trillian/types"
+)
+
+type intSelector func(prng *rand.Rand) int
+
+// validReadOps performs valid read operations against the map.
+type validReadOps struct {
+	vc              *client.MapClient
```

Comment:
This name is a bit outdated probably, as there is no "v" in the type name. How about `mc`?

Reply:
Done.